usagov/vote-gov-drupal
# VOTE-2507 remove content translator role

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2507

## Description

Remove Content Translator role from system

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/admin/people/roles and confirm there is no Content Translator role

## Checklist for the Developer




## Checklist for the Peer Reviewers
